### ISSUE TITLE
Can't bind to 'options' since it isn't a known property of 'input'

### ISSUE THREAD

**Issue by jimkeecn** [nbfontana/ngx-currency#198]
<mat-form-field>
      <mat-label>DRP</mat-label>
      <input matInput 
             formControlName="drp" autocomplete="off" currencyMask [options]="{prefix: '', suffix: '% ', decimal: '.' }" min="0" max="100">
      <mat-error *ngIf="drpForm.get('drp').status == 'INVALID'">
        DRP(0% - 100%)
      </mat-error>
    </mat-form-field>
    
    In here, it shows the error as above however, I do have some other similar input however, it does not throw error only in one page where is a html page used by a MatDialog.
    
    I have no idea what causes this.

**Comment by ArtemDintecom** [comment]
@jimkeecn I will assume that you deleted the non-existent NgxCurrencyModule, and forgot to import NgxCurrencyDirective.

**Comment by jimkeecn** [comment]
> @jimkeecn I will assume that you deleted the non-existent NgxCurrencyModule, and forgot to import NgxCurrencyDirective.

Do you mean Import like as documentation?

`import { NgxCurrencyDirective } from "ngx-currency";

@Component({
  standalone: true,
  selector: "app-root",
  templateUrl: "./app.component.html",
  imports: [NgxCurrencyDirective],
})`

However, My component is not a standalone component. I cannot change that component to a standalone component.

**Comment by ArtemDintecom** [comment]
@jimkeecn No, I mean import NgxCurrencyDirective into your dialog or page module, not into the AppModule, for sure they are different modules. You don't need to change your component to standalone for this.
